.github/workflows/deploy.yml

```
@@ -0,0 +1,101 @@
+name: Deploy
+on:
```

YAMLlint (1.38.0)

[warning] truthy value should be one of [false, true]

(truthy)